docs/modules/ROOT/pages/type-definitions/schema-configuration/type-configuration.adoc

```diff
@@ -0,0 +1,234 @@
+[[schema-configuration-type-configuration]]
+= Type Configuration
+
+A GraphQL Object type, representing a Neo4j Node, produces multiple operation fields in the `Query`, `Mutation`, and `Subscription` types, for example:
```

Comment:
Is "node" here the name of something (like a product or a tool) or is it just a regular node? If the latter is correct, then we should opt for lower case.

```suggestion
When representing a Neo4j Node, a GraphQL Object type produces multiple operation fields in the `Query`, `Mutation`, and `Subscription` types. 
For example:
```